### PR TITLE
add jobId to batch operation form

### PR DESCRIPTION
**Pull request by rossedfort**
## Description & motivation 💭 
Add an input to the batch operation form so that users can specify a Job ID for their batch operations. Fall back to a uuid if not specified.
### Screenshots (if applicable) 📸 
Screenshot 2023-10-09 at 3 21 37 PM
Screenshot 2023-10-09 at 3 22 13 PM

### How was this tested 👻 

- [x] Manual testing
- [ ] E2E tests added
- [ ] Unit tests added

### CODE REVIEW

File: src/lib/components/workflow/batch-operation-confirmation-modal.svelte [hunk 1]
```diff
@@ -97,17 +101,24 @@
...
+          'batch-operation-confirmation-input-hint',
+          { placeholder },
+        )}
+        {placeholder}
+      />
+      <Input
+        id="batch-operation-job-id"
+        label={translate('job-id')}
+        hintText={translate('batch', 'job-id-input-hint')}
+        bind:value={jobId}
```
Comment (rossedfort) [inline src/lib/components/workflow/batch-operation-confirmation-modal.svelte:119]: TODO: validate this is URL-safe, as the jobID ends up in the URL

**Review by Alex-Tideman — APPROVED** [review]

**Comment by Alex-Tideman** [comment]
👍 Nice

File: src/lib/components/workflow/batch-operation-confirmation-modal.svelte (hunk 1 above)
Comment (laurakwhit) [reply src/lib/components/workflow/batch-operation-confirmation-modal.svelte:131]: Can we use the utils in `encode-uri.ts` for this? Maybe add something to `routeForBatchOperation` real quick?

Comment (rossedfort) [reply src/lib/components/workflow/batch-operation-confirmation-modal.svelte:131]: Gah sorry I missed this before merging. I suppose we could have done that but I think this is also a totally reasonable approach.

**Pull request merged**